tests/e2e/tests/02-enable-serve-via-traffic/00-setup.yaml

```
@@ -0,0 +1,8 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: ../../manifest/setup.sh --manifest-dir ../../manifest
+    namespaced: true
+  - command: kubectl apply -f ../../manifest/global/pod-curl-target-gw.yaml -n default
+  - command: kubectl wait --for=condition=Ready pods -l app=curl-target-gw -n default
+    
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)